### PR TITLE
Helm upgrade v2->v3

### PR DESCRIPTION
**Pull request by jcortejoso**
### Description
Changes to support helm v3.x versions.
Details about the helm v2->v3 changes [in this link](https://helm.sh/docs/topics/v2_v3_migration/).

### Tested

Tested deploying a new cluster and some of the celotool packages.

### Backwards compatibility

Local system needs to have installed as `helm` command the helm v3.3.x version.

### CODE REVIEW

File: packages/helm-charts/blockscout/templates/blockscout-web.ingress.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
-apiVersion: networking.k8s.io/v1beta1
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+apiVersion: extensions/v1beta1
```
Comment (tkporter) [inline packages/helm-charts/blockscout/templates/blockscout-web.ingress.yaml]: Should this be `networking.k8s.io/v1` (& elsewhere)?

File: packages/celotool/src/lib/helm_deploy.ts [hunk 2]
```diff
@@ -895,3 +867,19 @@ export async function deleteFromCluster(celoEnv: string) {
...
+  const helmOK = await outputIncludes('helm version -c --short', requiredHelmVersion, `Checking local Helm version. Required ${requiredHelmVersion}`)
+  if (helmOK) {
+    return true
+  } else {
+    console.error(`Error checking local helm version. Helm version required ${requiredHelmVersion}`)
+    process.exit(1)
+  }
+}
+
+// export async function execHelmCmd<T>(exec: () => T): Promise<T> {
```
Comment (tkporter) [inline packages/celotool/src/lib/helm_deploy.ts]: Delete?

File: packages/helm-charts/cert-manager-cluster-issuers/requirements.lock [hunk 3]
```diff
@@ -3,4 +3,4 @@ dependencies:
   repository: https://charts.jetstack.io
   version: v0.11.0
 digest: sha256:51683512f39cf91681ecfb04428c086da8cdaf29f27d39e82c13052464a69beb
-generated: "2020-09-10T16:24:26.915183-04:00"
+generated: "2020-10-21T13:02:03.998411023+02:00"
```
Comment (tkporter) [inline packages/helm-charts/cert-manager-cluster-issuers/requirements.lock:6]: does this file also get deleted?

**Review by tkporter — APPROVED** [review]
LGTM, I think we should hold off on merging this until all clusters have been upgraded and people are aware they need helm v3.3

File: packages/celotool/src/lib/helm_deploy.ts (hunk 2 above)
Comment (jcortejoso) [reply packages/celotool/src/lib/helm_deploy.ts]: Yep

File: packages/helm-charts/blockscout/templates/blockscout-web.ingress.yaml (hunk 1 above)
Comment (jcortejoso) [reply packages/helm-charts/blockscout/templates/blockscout-web.ingress.yaml]: Right. I was testing some issues with helm 3 and I forced the apiVersion.

File: packages/helm-charts/cert-manager-cluster-issuers/requirements.lock (hunk 3 above)
Comment (jcortejoso) [reply packages/helm-charts/cert-manager-cluster-issuers/requirements.lock:6]: The `requirements.yaml` now is part of `Chart.yaml`, but the `requirements.lock` is keeped.
I think it is safe to add them to gitignore.

**Pull request merged**